wiki/People/Tournament_Committee/en.md

```diff
@@ -26,7 +33,111 @@ Their responsibilities include:
 - Regularly reviewing the [Official beatmapping contest support](/wiki/Contests/Official_support) page for revisions when necessary
 
-The panel was first announced to the public on February 28, 2024 through the [Mapping Discussion](https://osu.ppy.sh/community/forums/topics/1890429?n=1) forum.
+The panel was first announced to the public on February 28, 2024 through the [Mapping Discussion](/community/forums/topics/1890429) forum.
+
+## Tournament Committee Workflow
+
+### Tournament Reports
+
+Tournament reports are the preferred way for players and tournament staff alike to report infringements of the Official Support expectations and eligibility. When a new report is sent in, a new thread about that report is automatically created and the tournament committee team is automatically pinged. Members will read the report and determine whether to pursue action based on the report contents. Such action includes but is not limited to:
+
+- Noting the tournament for investigation pending further reports if this report alone is not enough to warrant further action
+- Further investigation and context gathering
+- Reaching out to the infringing party for further information about the events in the report
+- Voting on whether to withhold a tournament badge
+- Voting on whether to hand out a tournament ban
+- Voting on whether to hand out a tournament staffing ban
+
+### Tournament Review
+
+Assuming the tournament has submitted a request for screening in a timely manner, removed any disallowed players from the tournament, and sent a conclusion email with the badge attached, the Tournament Committee will assign two random members to review the tournament before confirming its badge. The assigned members will ensure the following:
+
+- The format is eligible for badge
+- The tournament theme does not violate community rules
+- Any signup restrictions are geographically and/or culturally consistent
+- Any unofficial screening process is open-source with a publicly available dataset
+- All results are up-to-date on both the tournament’s bracket and any public-facing documents, such as a spreadsheet
+- All multiplayer lobby links are publicly available
+- Staff and player lists are publicly available
+- The badge design does not contain any art owned by a third party
+
+Assuming the tournament meets all these checks, the assigned members will mark the tournament as approved for badging, and the account support team will apply the badge at a later time to the winner(s) profile(s). If the tournament does not meet one or more of these checks, the assigned members will ask the account support team to request any necessary changes from the tournament organiser(s) to meet these checks before marking the tournament as approved for badging. If these changes are not possible after the tournament’s conclusion, the tournament committee will hold an anonymous vote as to whether to approve or deny a badge for the tournament.
```

Comment:
perhaps mention that the assigned members are required to state any conflict of interest (eg. staffing/playing in that tournament or being close to people who are), and get someone else randomly assigned in that scenario